# Program to throw an exception with a custom message

def throw_exception_with_custom_message():
    # Raising a generic exception with a custom message
    raise Exception("This is a custom exception with my own message!")

# Calling the function to throw the exception
throw_exception_with_custom_message()
